### PR TITLE
Adds pangram exercise

### PR DESCRIPTION
**Pull request by amscotti**
Hi, this PR is to add a new exercise for `pangram`, [here is a link to the problem-specifications](https://github.com/exercism/problem-specifications/tree/master/exercises/pangram)

This is one of the exercises on the list in issue #149
 
I'm still learning Dart, so feel free to point out anything that needs to be change or improved.

One note, this may need to be rebased if #172 is merged due to conflicts in the `config.json` file.

Thanks! :)

### CODE REVIEW

**Comment by amscotti** [comment]
Ah, looks like there is an issue with this code for Dart 1.24.3. I'll look at updating this for it works for both Dart 2 and 1.

**Comment by amscotti** [comment]
Updated the code, and all test pass for both Dart 1 and 2.

File: exercises/pangram/lib/example.dart [hunk 1]
```diff
@@ -0,0 +1,9 @@
+class Pangram {
+  bool isPangram(String sentence) {
```
Comment (amscotti) [inline exercises/pangram/lib/example.dart]: is Dart, is it better to have the name like `isPangram` or `is_pangram`?

Comment (jvarness) [reply exercises/pangram/lib/example.dart]: `isPanagram` is the preferred spelling :)

https://www.dartlang.org/guides/language/effective-dart/style#do-name-other-identifiers-using-lowercamelcase

**Comment by jvarness** [comment]
@amscotti thanks for submitting your first PR!! @Stargator is the owner of the repository, and he will provide guidance for the `config.json`. He, along with myself and @devkabiir work on many of the exercises here, and we typically all review each others submissions before merging.

We'll get it reviewed and provide feedback :) thanks for your contributions!

**Comment by jvarness** [comment]
Just looked at our open PRs and realized this isn't your first PR :) still, thanks for contributing!

File: config.json [hunk 2]
```diff
@@ -242,6 +242,19 @@
         "lists",
         "math"
       ]
+    },
+    {
+      "slug": "pangram",
+      "uuid": "0ea556b0-afcd-479d-bbad-68bc4556c1d0",
+      "core": false,
+      "unlocked_by": null,
```
Comment (Stargator) [inline config.json]: I think having this unlocked after anagram would fit well, what do you think?

Comment (amscotti) [reply config.json]: Yeah, that seems like a good idea to me. I'll make the update!

**Pull request merged**